TEAMS.md

```diff
@@ -0,0 +1,59 @@
+# Projects details
+
+| *Project*       | *Maintainers*  | *Teams* |
+| --------------- | -------------- | ------- |
+[ZendSkeletonApplication](https://github.com/zendframework/ZendSkeletonApplication) | [Akrabat](https://github.com/akrabat), [EvantDotPro](https://github.com/EvanDotPro) | community-review-team |
```

Comment:
@zendframwork/{team name} creates a broken link :(
If I use @username works in github but if you use it outside this site doesn't work..